variables-skyline/index110.html

```diff
@@ -63,5 +63,5 @@
 </html>
 
 <!--
-  To use a variable, just put the variable name in parenthesis with `var` in front of them like this: `var(--variable-name)`. So whatever value you gave the variable will now be applied to whatever property you use it on. Add your variable as the value of the `background-color` property of the `bb1a` class.
+  To use a variable, just put the variable name in parenthesis with `var` in front of them like this: `var(--variable-name)`. Add your variable as the value of the `background-color` property of the `bb1a` class. Whatever value you gave the variable will be applied to whatever property you use it on. In this case, your variable has the value of `#999`. So `#999` will be used as the value for the `background-color` property.
```

Comment:
This is a great description, but could be shortened slightly: "To use a variable, just put the variable name in parenthesis with `var` in front of them like this: `var(--variable-name)`. Add your variable as the value of the `background-color` property of the `bb1a` class. Whatever value you gave the variable will be applied to whatever property you use it on, so `#999` will be used as the value for the `background-color` property."